Assets/HoloToolkit/Input/Tests/Scripts/ChangeColorOnFocus.cs

```diff
@@ -0,0 +1,43 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using HoloToolkit.Unity;
+using UnityEngine;
+
+/// <summary>
+/// Controls object appearance by changing its color when focused.
+/// </summary>
+public class ChangeColorOnFocus : Interactable
```

Comment:
This feels like a test script showing usage. Any reasons for this it's in the Scripts folder?